torch/optim/lbfgs.py

```diff
@@ -101,7 +101,7 @@ def step(self, closure):
 
         # evaluate initial f(x) and df/dx
         orig_loss = closure()
-        loss = orig_loss.data[0]
+        loss = float(orig_loss)
```

Comment:
why not item?  Is the thought that float will be slightly faster?

Reply:
Same speed. I've had a small preference for float()/int() since they work in 0.3 (not particularly important here, but matters for changes to benchmarks/examples).